utils/db_test.ts

```diff
@@ -205,3 +208,49 @@ Deno.test("[db] createComment() + getCommentsByItem()", async () => {
   await assertRejects(async () => await createComment(comment2));
   assertArrayIncludes(await getCommentsByItem(itemId), [comment1, comment2]);
 });
+
+Deno.test("[db] compareScore()", () => {
```

Comment:
Another way we could do this is to create an items array in the wrong order by score, then sort it via `compareScore()` and check the new order. This doesn't improve anything in a technical sense but does convey the intent and behaviour of the use of `compareScore()` to the reader. WDYT?

Reply:
Yes, that sounds like a good idea! Will implement it towards the end of the week :)

Reply:
No worries!

Reply:
Updated it in [153f423](https://github.com/denoland/saaskit/pull/300/commits/153f42317f81284fe78d366bc4df643e9ed3afce). This is way better than before. I also added a third item to not just switch the items from left to right.